UsingSimpleSqlProvider/app/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion compileSdkVer
+    buildToolsVersion buildToolsVer
+    defaultConfig {
+        applicationId "me.gurpreetsk.usingsimplesqlprovider"
```

Comment:
Package name should follow standards as mentioned in [Pull Request guide](https://github.com/nisrulz/android-examples#pull-requests)